src/js/views/forms/form/form_views.js

```diff
@@ -34,9 +34,7 @@ const ContextTrailView = View.extend({
     'click .js-dashboard': 'click:dashboard',
   },
   onClickBack() {
-    Radio.request('history', 'go:back', () => {
-      this.routeToPatient();
-    });
```

Comment:
Pertaining to this coverage miss: https://coveralls.io/builds/58331475/source?filename=src%2Fjs%2Fviews%2Fforms%2Fform%2Fform_views.js#L38.

Does `this.routeToPatient()` do anything here? Testing in my local sandbox, this behaves the same without it.

If that can be removed here, we should be able to change `onClickDashboard()` below to this:

```js
onClickDashboard() {
  Radio.trigger('event-router', 'patient:dashboard', this.patient.id);
},
```

And get rid of the `routeToPatient()` function.

Reply:
This was added so if you hard refreshed on a form and did not have any previous history, the back button would go to the patient.  It's needed.. I dunno how we'd actually test it though.

Reply:
Ah okay, I just added that back.

`istanbul ignore`?